core-blocks/playlist/index.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+
+export const name = 'core/playlist';
+
+export const settings = {
+	title: __( 'Playlist' ),
+
+	description: __( 'The Playlist block allows you to embed playlist files and play them back using a Core playlist.' ),
+
+	icon: 'format-audio',
```

Comment:
Should we change the icon to `dashicons-playlist-video`?
https://developer.wordpress.org/resource/dashicons/#playlist-video

Reply:
I love the idea of that. I struggled to decided an icon and the only reason I went with the audio is because technically both video and audio can produce sound. haha

`dashicons-playlist-video` makes so much more sense. Great suggestion!